test/vaulttracker/VaultTracker.sol

```diff
@@ -246,6 +211,14 @@ contract VaultTracker {
     return true;
   }
 
+  /// @notice Return both the current maturityRate if it's > 0 (or exchangeRate in its place) and the Compounding exchange rate
+  /// @dev While it may seem unnecessarily redundant to return the exchangeRate twice, it prevents many kludges that would otherwise be necessary to guard it
+  /// @return maturityRate, exchangeRate if maturityRate > 0, exchangeRate, exchangeRate if not.
+  function rates() public view returns (uint256, uint256) {
+    uint256 exchangeRate = Compounding.exchangeRate(protocol, cTokenAddr);
+    return maturityRate > 0 ? (maturityRate, exchangeRate) : (exchangeRate, exchangeRate);
```

Comment:
I noticed we use these operators `?` etc. instead of `if-else` statements. If they aren't more efficient, I think we should make it a practice to avoid these ternary operators as they are a bit harder to read. That said, this is a nit and something that can be addressed another time.

Reply:
ternary is the correct operator for a truly binary decision, so i support their use.

effeciency would be a determining factor but i'm pretty sure it results in the same bytecode. could be worth a look.

Reply:
Im neutral on this overall.

1. I agree its slightly harder to read which can be important in solidity where were throwing it at auditors
2. But those same auditors have expectations based on lines of code submitted and semi charge based on that so there is incentive to reduce LOC overall lol